lib/ansiblelint/color.py

```diff
@@ -1,6 +1,17 @@
-"""Console coloring support."""
+"""Console coloring and terminal support."""
 from enum import Enum
 
+from rich.console import Console
+from rich.theme import Theme
+
+_theme = Theme({
+    "info": "cyan",
+    "warning": "dim yellow",
```

Comment:
Have you tested it in standard-compliant terminal emulators like Kitty? I'm pretty sure this will explode (but needs checking).

Reply:
Tested with kitty too, worked without any exploding kitterns.